actions/pypi_build/action.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
This file is already removed from latest main. Can you please rebase from latest main before merging?

Reply:
yeah, it showed locally as deleted when i updated.